README.md

```diff
@@ -1 +1,36 @@
-# syncer
\ No newline at end of file
+# action-sync-fork
+
+Syncs galaxy integration forks with upstream
+
+## Example Usage
+
+[`.github/workflows/sync.yml`](https://github.com/GOG-Nebula/galaxy-riot-integration/blob/9da15bdfd5f30f2aa6e887491d7f53d0853adeae/.github/workflows/sync.yml)
```

Comment:
outdated content. Lets remove and keep only the example below

Reply:
done